packages/react/src/components/dropdown/select/Select.tsx

```diff
@@ -116,7 +117,7 @@ export type CommonSelectProps<OptionType> = {
   /**
    * The label for the dropdown
    */
-  label: React.ReactNode;
+  label?: React.ReactNode;
```

Comment:
Could we have some guidance or typing that component requires either label or aria-labelledby?

Reply:
- [ ] Add typing for conditionally requiring eitgher label or aria-labelledby